packages/@o3r/configuration/src/devkit/configuration-devtools.contextualization.module.ts

```diff
@@ -0,0 +1,34 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { StoreModule } from '@ngrx/store';
+import { ConfigurationStoreModule } from '../stores/index';
+import type { ConfigurationContextualizationServiceOptions } from './configuration-devtools.interface';
+import { OTTER_CONFIGURATION_CONTEXTUALIZATION_DEFAULT_OPTIONS, OTTER_CONFIGURATION_CONTEXTUALIZATION_OPTIONS } from './configuration-devtools.token';
+import { ConfigurationDevtoolsContextualizationService } from './configuration-devtools.contextualization.service';
+
+@NgModule({
+  imports: [
+    StoreModule,
+    ConfigurationStoreModule
+  ],
+  providers: [
+    { provide: OTTER_CONFIGURATION_CONTEXTUALIZATION_OPTIONS, useValue: OTTER_CONFIGURATION_CONTEXTUALIZATION_DEFAULT_OPTIONS },
+    ConfigurationDevtoolsContextualizationService
+  ]
+})
+export class ConfigurationContextualizationDevtoolsModule {
+
+  /**
+   * Initialize Otter Devtools
+   *
+   * @param options
+   */
+  public static instrument(options: Partial<ConfigurationContextualizationServiceOptions>): ModuleWithProviders<ConfigurationContextualizationDevtoolsModule> {
```

Comment:
could turn `options` optional to avoid the `.instrument({})` in the app

Reply:
If you don't need to pass options, then you don't need to call instrument, importing the module will be enough

Reply:
but it is what you do in the showcase app